sample/HttpTrigger/function.json

```diff
@@ -2,7 +2,8 @@
   "bindings": [
     {
       "type": "httpTrigger",
-      "direction": "in"
+      "direction": "in",
+      "methods": [ "get" ]
```

Comment:
The BAT example in the e2e tests doesn't have a methods specification. I've tested it both ways.